include/_mgp.hpp

```diff
@@ -236,6 +238,56 @@ inline mgp_type *type_nullable(mgp_type *type) { return MgInvoke<mgp_type *>(mgp
 
 // mgp_graph
 
+inline bool CreateLabelIndex(mgp_graph *graph, const std::string_view label) {
```

Comment:
These names are not consistent with namings in file, can you change that to `create_label_index`?

Reply:
Yep good catch